frontend/pages/hosts/details/cards/Software/SoftwareDetailsModal/SoftwareDetailsModal.tests.tsx

```diff
@@ -1,137 +0,0 @@
-import React from "react";
```

Comment:
Can create an `InventoryVersionsModal.tests.tsx` and move these tests onto there? Or we can make unit tests for `InventoryVersions.tsx`

Reply:
good idea